flink-scala-shell/src/test/scala/org/apache/flink/api/scala/ScalaShellITCase.scala

```diff
@@ -23,7 +23,7 @@ import java.io._
 import akka.actor.ActorRef
 import akka.pattern.Patterns
 import org.apache.flink.runtime.minicluster.StandaloneMiniCluster
-import org.apache.flink.configuration.{ConfigConstants, Configuration, CoreOptions, GlobalConfiguration}
+import org.apache.flink.configuration._
```

Comment:
I think in this case we can avoid the wildcard import.

Reply:
make sense.